testing/fceux/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Danct12 <danct12@disroot.org>
+# Maintainer: Danct12 <danct12@disroot.org>
+pkgname=fceux
+pkgver=2.2.3
+pkgrel=0
+pkgdesc="Fast and ultra-compatible NES/Famicom emulator"
+url="http://www.fceux.com"
+arch="all"
+license="GPL"
+depends="sdl gd gtk+2.0"
+depends_dev="sdl-dev gd-dev gtk+2.0-dev scons mesa-dev glu-dev"
+makedepends="$depends_dev"
+install=""
+subpackages="$pkgname-doc"
+source="https://downloads.sourceforge.net/project/fceultra/Source%20Code/$pkgver%20src/$pkgname-$pkgver.src.tar.gz
+	scons-python3.patch"
+options="!check" # no testsuite
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
```

Comment:
Why define prepare if you're going to define it as the default definition ?

remove this